src/interface/file.rs

```diff
@@ -49,6 +49,10 @@ pub fn removefile(filename: String) -> std::io::Result<()> {
     Ok(())
 }
 
+fn is_allowed_char(c: char) -> bool{
+    char::is_alphanumeric(c) || c == '.'
```

Comment:
This is a good change. I simplified it to just alphanumeric on my first iteration. Obviously we need periods for usual filenames. There's probably a few more characters we can allow. Not sure if its worth doing now.